src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -363,23 +364,40 @@ export class NotebookModel extends Disposable implements INotebookModel {
 						}
 					});
 				}
+				// Get Notebook URI Params & adjust to string
+				let notebookUriParams: string = this.notebookUri.query;
+				// Format query to code cell format
+				notebookUriParams = notebookUriParams.replace(/&/g, '\n').replace(/=/g, ' = ');
```

Comment:
You should test what happens when the parameters have & or = in them (both key and value). Ideally you should be able to have those values be % encoded before hand and then decode them here - but given what we found here we may not be able to do this without core changes. But at the very least you should test this out and see what the behavior is so we can at least document it.

Reply:
Interesting, are you saying something like `...Input.ipynb?x=1&1&y=2`. 
I believe that this is not a valid scenario for parameterization. Moreover, would be an extreme edge case in which we would not expect users to ever use **&** within parameters.

Reply:
Why is that not a valid scenario? I would expect string parameters to be supported as well. And I wouldn't necessarily call an & in a string an "extreme edge case".

But that's up to you to determine if you want to support. Just make sure if you don't it's clearly documented. And I would suggest bringing this up with the other stakeholders in this spec so they're aware what is/isn't supported.

Reply:
And generally I would suggest that if you do have edge cases like this that you handle in certain ways you add tests for those cases. This shows

1. That the behavior is as you expect when you wrote the code originally
2. People can clearly see what is supported and how it should behave in case they're making changes later

Reply:
Yup, I think that would be a difficult edge case to solve as well, `...Input.ipynb?x=1&1&y=2`. Since we specifically use this format with **&** as a separator for Notebook Parameters we would treat them as a new parameter.
As such, I will be sure to document this. I can create more test cases to ensure that this is the behavior we expect.